clang/docs/ReleaseNotes.rst

```diff
@@ -493,6 +493,9 @@ Bug Fixes in This Version
   Fixes (`#65143 <https://github.com/llvm/llvm-project/issues/65143>`_)
 - Fix crash in formatting the real/imaginary part of a complex lvalue.
   Fixes (`#69218 <https://github.com/llvm/llvm-project/issues/69218>`_)
+- No longer use C++ ``thread_local`` semantics in C23 when using
```

Comment:
Does this patch also help with https://github.com/llvm/llvm-project/issues/69167 ?

Reply:
It does, thank you for spotting that! I'm going to add another test case from that issue when landing this.